Campaign/resources/views/banners/_form.blade.php

```diff
@@ -27,6 +27,8 @@
 
     remplib.bannerForm.bind("#banner-form", banner);
 
+    var clientSiteUrl = '{{ Config::get('app.client_site_url') }}';
```

Comment:
If `clientSiteUrl` is not set, user is redirected to same URL with fragment identifier bannerPicker. No message with info or error what happend. Maybe we could hide (or disable) button if `clientSiteUrl` is empty?